dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java

```diff
@@ -42,32 +44,37 @@ public Status sendTweets(String tweet) throws TwitterException {
     }
 
 
-    public ArrayList<TweetRespons> fetchLatestTweets() {
-
-        ArrayList<TweetRespons> arrayList = new ArrayList<>();
-        List<Status> statuses = null;
+    public ArrayList<TweetResponse> fetchLatestTweets() {
+        ArrayList<TweetResponse> TweetList = new ArrayList<>();
         try {
-            statuses = twitter.getHomeTimeline();
+            List<Status> statuses = twitter.getHomeTimeline();
             for (int i = 0; i < statuses.size(); i++) {
-                String twitterHandle;
-                String name;
-                String message = null;
-                String profileImageUrl = null;
                 Date createdAt = null;
                 Status status = statuses.get(i);
-                message = status.getText();
-                name = status.getUser().getName();
-                profileImageUrl = status.getUser().getProfileImageURL();
+                String message = status.getText();
+                String name = status.getUser().getName();
+                String profileImageUrl = status.getUser().getProfileImageURL();
                 createdAt = status.getCreatedAt();
                 Format format = new SimpleDateFormat("dd-mm-yyy HH:mm:ss");
                 String date = format.format(createdAt);
-                twitterHandle = status.getUser().getScreenName();
-                tweetRespons = new TweetRespons(message, name, twitterHandle, profileImageUrl, date);
-                arrayList.add(tweetRespons);
+                String twitterHandle = status.getUser().getScreenName();
+                tweetResponse = new TweetResponse(message, name, twitterHandle, profileImageUrl, date);
+                TweetList.add(tweetResponse);
             }
         } catch (TwitterException e) {
+            logger.error("error in fetching tweets");
         }
-        return arrayList;
+        return TweetList;
+    }
+
+    public List<TweetResponse> getFilteredTweets(String tweets) {
+        ArrayList<TweetResponse> listTweets = fetchLatestTweets();
+        int len=tweets.length();
```

Comment:
here align code properly 

cmd + options +shift + L , when pressed at the same time, u will get code alignment window , run it please

Reply:
addressed the review